tj2-reco.py

```diff
@@ -82,6 +84,18 @@ def add_pixelmaskers(path):
   
   return path
 
+def add_pixel_calibration(path):
+  pixcal = Processor(name="PixelChargeCalibrator",proctype="PixelChargeCalibrator")
+  pixcal.param('SparseDataCollectionName', "zsdata_tj2_raw")
+  pixcal.param('CalibratedCollectionName', "zsdata_tj2")
+  pixcal.param('GainCalibrationDBFileName', "/home/benjamin/tbsw_workspace_tjmp2_desy/calibrationDBFile.root")
```

Comment:
this hardcoded path needs to be adjustet. maybe read it from command line with Argparse. 
or execute calibFuncExample.py on the file in this script with a known path. 

but hardcoding like above will work for the moment.